### ISSUE TITLE
Running dotnet nuget locals --clear global-packages on linux, causes dotnet cache priming failure in subsequent runs

### ISSUE THREAD

**Issue by mishra14** [NuGet/Home#3316]
Creating an issue to track the behavior on linux.

Repro [Once, dotnet nuget locals command is lit up] - 
1. Run `dotnet nuget locals --clear global-packages`
2. Run `dotnet nuget locals --list all`
3. Observe that dotnet is not able to prime the nuget cache.

**Comment by mishra14** [comment]
The behavior is not observed on Mac or Windows. The dotnet cli team has mentioned that it could be due to case sensitivity on linux. 
They have a fix on feature/msbuild branch, which will be merged into rel/1.0.0 within a week. 

Needs more investigation once the feature/msbuild is merged into rel.

**Comment by mishra14** [comment]
Retested after feature/msbuild was merged into rel/1.0.0. The error is no longer observed.
